astroquery/query.py

```diff
@@ -203,6 +203,10 @@ def __init__(self):
 
         self.name = self.__class__.__name__.split("Class")[0]
 
+    def __call__(self, *args, **kwargs):
+        """ init a fresh copy of self """
+        return self.__class__(*args, **kwargs)
+
```

Comment:
This is a global change to all modules and is not the intended behavior.  

Perhaps we need to clarify this, but the behavior I'd expect as a user is:
```python
orig = SimbadClass()
fewer_rows = orig(ROW_LIMIT=10)
extra_thing = fewer_rows(table_columns=['**'])
```
would result in `orig` having the default row limit and both `fewer_rows` and `extra_thing` having `ROW_LIMIT=10`.

In any case, if a change in this behavior is to be made, I would like it in a separate PR

Reply:
Is it? I was mirroring what is done in `BaseQuery` here https://github.com/astropy/astroquery/blob/25e6521ad3e86f5b2285d08e736cd3417159482d/astroquery/query.py#L223 

Otherwise this previous behavior of the Simbad module (which was a child of `BaseQuery` before) would not work:

```python
from astroquery.simbad import Simbad
Simbad.query_object("M1")
```

```
TypeError: 'SimbadClass' object is not callable
```

but one would have to do 

```python
from astroquery.simbad import Simbad
Simbad().query_object("M1")
```

which I'm not opposed to but **is** an API change

Reply:
A suggestion to keep the previous behavior without changing `BaseVOQuery` could be to define the `__call__` method in `SimbadClass` instead of in the `BaseVOQuery` one? What do you think?

Reply:
I'm sorry, it looks like you're correct and I got it wrong.

I'm OK with including this change here.